helm.tf

```diff
@@ -99,191 +82,830 @@ resource "helm_release" "metrics_server" {
 
 }
 
+resource "helm_release" "cert_manager" {
+  count      = var.helm_cert_manager_enabled || var.k8s_opentelemetry_enabled ? 1 : 0
+  name       = "cert-manager"
+  namespace  = "cert-manager"
+  repository = "https://charts.jetstack.io"
+  chart      = "cert-manager"
+  version    = "1.6.1"
```

Comment:
We should allow chart version selection (module input)

Reply:
I'm marking this as resolved because we agreed to don't do it